product-service/src/main/java/com/extremecoder/productservice/exception/GlobalExceptionHandler.java

```diff
@@ -141,23 +141,21 @@ public ResponseEntity<Object> handleBindException(BindException ex) {
         // todo: modify response
         log.error(ex.getMessage());
 
-        Map<String, Object> body = new HashMap<>();
 
         //Get all errors
         List<String> errors = new ArrayList<>();
 
-        for (ObjectError allError : ex.getBindingResult().getAllErrors()) {
-            if (allError instanceof FieldError) {
-                errors.add(((FieldError) allError).getField() + " " + allError.getDefaultMessage());
-            } else if (allError instanceof ObjectError) {
-                errors.add(allError.getDefaultMessage());
+        ex.getBindingResult().getAllErrors().forEach(objectError -> {
+            if (objectError instanceof FieldError) {
+                errors.add(((FieldError) objectError).getField() + " " + objectError.getDefaultMessage());
+            } else if (objectError != null) {
```

Comment:
I think this logic should be covered earlier. Because if `objectError` is null, it will through null pointer exception in line: 149.